tests/acceptance/cli_test.py

```diff
@@ -1244,6 +1244,13 @@ def test_up_with_net_v1(self):
         assert foo_container.get('HostConfig.NetworkMode') == \
             'container:{}'.format(bar_container.id)
 
+    def test_up_ordered_networks(self):
+        self.base_dir = 'tests/fixtures/networks'
+        result = self.dispatch(['-f', 'ordered-networks.yml', 'up', '-d'])
+
+        assert 'Connecting to networks_buzz\nConnecting to networks_foo' \
```

Comment:
If we're removing the `print` statement above, you'll at least need to add `--verbose` to the `dispatch` call above. 
But I'd be wary of relying on those 3 specific lines appearing consecutively all the time. I think a better test would involve different `mac_address` values, and checking that the network with the highest priority sets the `mac_address` for the container.
If you do it that way, it's also probably easier to do it as an integration test so you can change priority values and check the container configuration changes accordingly.